### PR TITLE
Fix linting on CHC

### PR DESCRIPTION
**Pull request by chinandrew**
Summary of changes:
- Fix linting issues in CHC. Mainly long lines and formatting log strings

Note that there are still linting issues after this PR that will be closed by #431 and #460

### CODE REVIEW

File: changehc/delphi_changehc/update_sensor.py [hunk 1]
```diff
@@ -235,30 +242,4 @@ def update_sensor(self,
...
-        print(fails)
-
-        # Filter existing exports to exclude those that failed to archive
-        succ_common_diffs = {f: diff for f, diff in common_diffs.items() if f not in fails}
-        arch_diff.filter_exports(succ_common_diffs)
-
-        # Report failures: someone should probably look at them
-        for exported_file in fails:
-            print(f"Failed to archive '{exported_file}'")
-        '''
```
Comment (chinandrew) [inline changehc/delphi_changehc/update_sensor.py:264]: not sure if these were still needed

Comment (rumackaaron) [reply changehc/delphi_changehc/update_sensor.py:264]: I'm not sure how exactly we use S3 and archiving. @krivard?

Comment (krivard) [reply changehc/delphi_changehc/update_sensor.py:264]: Each day when the indicator runs, it produces some output. Much of that output may match what's already in the API. To conserve disk space and memory, we want to exclude those matching rows from the new issue to be uploaded to the API. The easiest way to do this is to compare the daily output with a set of CSV files matching the current (most recent issue for each day of data) contents of the API. These files are kept in a cache in a versioned S3 bucket. After the day's output is produced, the archive differ updates the cached files, and edits the files in receiving to only include lines with updated or new data.

Comment (krivard) [reply changehc/delphi_changehc/update_sensor.py:264]: Probably best not to remove those lines until we have a fix to automation ready to run the archive differ as a subsequent process.

Comment (chinandrew) [reply changehc/delphi_changehc/update_sensor.py:264]: Just to be clear these are currently commented out with the triple single quotes, is it worth leaving them in as reference?

Comment (krivard) [reply changehc/delphi_changehc/update_sensor.py:264]: oh right! Change is new, so I suppose it makes sense that it's not already running the archive differ. 

In that case let's begin as we mean to go on, and drop them.

**Review by rumackaaron — APPROVED** [review]

**Pull request merged**